war/src/main/java/io/lumeer/engine/controller/ConfigurationFacade.java

```diff
@@ -329,52 +384,81 @@ public void resetUserConfiguration() {
     *       the name of attribute to remove
     */
    public void resetUserConfigurationAttribute(final String attributeName) {
-      resetConfiguration(true, attributeName);
+      resetConfiguration(ConfigurationLevel.USER, attributeName);
+   }
+
+   /**
+    * Resets active project configuration. The whole project configuration entry will not be deleted from the system collection, just the config field!
+    */
+   public void resetProjectConfiguration() {
+      resetConfiguration(ConfigurationLevel.PROJECT, null);
+   }
+
+   /**
+    * Removes the specified attribute located in project configuration entry.
+    *
+    * @param attributeName
+    *       the name of attribute to remove
+    */
+   public void resetProjectConfigurationAttribute(final String attributeName) {
+      resetConfiguration(ConfigurationLevel.PROJECT, attributeName);
    }
 
    /**
-    * Resets currently logged team configuration. The whole team configuration entry will not be deleted from the system collection, just the config field!
+    * Resets currently logged organisation configuration. The whole organisation configuration entry will not be deleted from the system collection, just the config field!
     */
-   public void resetTeamConfiguration() {
-      resetConfiguration(false, null);
+   public void resetOrganisationConfiguration() {
+      resetConfiguration(ConfigurationLevel.ORGANISATION, null);
```

Comment:
I do not like this approach. Why do we reset configuration by setting `null`? We can accidentally reset the configuration when `null` value is passed without us noticing it.